Chummer/data/gear.xml

```diff
@@ -1532,6 +1532,68 @@
       </weaponbonus>
       <ammoforweapontype>firefighting cannons</ammoforweapontype>
     </gear>
+	<gear>
+      <id>d71ea9c1-9b5f-428e-b849-7d665eb31c5e</id>
+      <name>Ammo: Sling-Shot Acid Capsules</name>
+      <category>Ammunition</category>
+      <rating>0</rating>
+      <source>SOTG</source>
+      <page>13</page>
+      <avail>-</avail>
+      <cost>100</cost>
+      <costfor>10</costfor>
+      <weaponbonus>
+        <damagereplace>6P Acid</damagereplace>
+      </weaponbonus>
+      <ammoforweapontype>slingshot</ammoforweapontype>
+    </gear>
+	<gear>
+      <id>e20f021b-1d00-4443-a481-d417114e067e</id>
+      <name>Ammo: Sling-Shot Paint Capsules</name>
+      <category>Ammunition</category>
+      <rating>0</rating>
+      <source>SOTG</source>
+      <page>13</page>
+      <avail>-</avail>
+      <cost>10</cost>
+      <costfor>10</costfor>
+      <weaponbonus>
+        <damagereplace>0</damagereplace>
+      </weaponbonus>
+      <ammoforweapontype>slingshot</ammoforweapontype>
+    </gear>
+	<gear>
+      <id>5ece577b-7c15-4f2c-9f4b-cd325154cf9c</id>
+      <name>Ammo: Sling-Shot Poison Capsules</name>
+      <category>Ammunition</category>
+      <rating>0</rating>
+      <source>SOTG</source>
+      <page>13</page>
+      <avail>-</avail>
+      <cost>100</cost>
+      <costfor>10</costfor>
+      <weaponbonus>
+        <damagereplace>As Narcoject</damagereplace>
+      </weaponbonus>
+      <ammoforweapontype>slingshot</ammoforweapontype>
+    </gear>
+	<gear>
+      <id>d82e68e5-08d1-4f27-af45-de54ad1dcd72</id>
+      <name>Ammo: Slingshot Capsule Round</name>
+      <category>Ammunition</category>
+      <rating>0</rating>
+      <source>RG</source>
+      <page>23</page>
+      <avail>-</avail>
+      <cost>5</cost>
+      <costfor>10</costfor>
+      <weaponbonus>
+        <damagereplace>As Substance Within</damagereplace>
```

Comment:
"As Substance Within" is a term that won't get translated into other languages (note, neither will "Acid" for an earlier slingshot entry). "As Drug/Toxin", "Special", or "Chemical" would get translated though.

Reply:
Changed to "As Drug/Toxin."